articles/cognitive-services/Face/face-how-to-install-containers.md

```diff
@@ -89,7 +89,7 @@ For a full description of available tags for the Face container, see [Recognize
 > [!TIP]
 > You can use the [docker images](https://docs.docker.com/engine/reference/commandline/images/) command to list your downloaded container images. For example, the following command lists the ID, repository, and tag of each downloaded container image, formatted as a table:
 >
->  ```Docker
```

Comment:
note: this still probably isn't a correct language block because this is actually a block of commands